src/documents/tutorial-prefsFrameworkMinimalFootprint/MinimalFootprint.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Preferences Framework: Minimal Footprint
+title: "Preferences Framework: Minimal Footprint"
```

Comment:
Why are there quotes here and not in other documents?

Reply:
Actually this was interfering in the YAML format of the frontmatter (more specifically, the ":"). Hence the qoutes.

Reply:
Thanks, good to know.

Reply:
Use `&#58;` in place of colon in title.

Reply:
This doesn't seem to work, the title shows literal `&#58;`.